.gitignore

```diff
@@ -1,3 +1,9 @@
-.pio
-.vscode
-platformio.ini
+.pioenvs
+.piolibdeps
+*/*
+.pio/*
+*.*
+**.*
+!*.c
+!*.h
+!platformio.ini
```

Comment:
Newline at end of file.

Reply:
wat matthijs zegt

Reply:
exclude je nou ook die hele vscode map enzo? ik snap al je include pointers niet dus